### PR TITLE
Updated Metrics Setup docs for the dotnet SDK to prepare for GA

### PR DESCRIPTION
**Pull request by jamescrosswell**
## DESCRIBE YOUR PR

Resolves https://github.com/getsentry/sentry-dotnet/issues/3245

When Metrics are GA, the syntax for enabling Metrics in the .NET SDK will change.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [x] Other deadline: Whenever Metrics are GA

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.

@bitsandfoxes I'm having difficulty with this one. I can't work out how to change what shows in the "ON THIS PAGE" pane... this doesn't appear to update automatically when I add new H1, H2 headers etc.

Once we're happy with the content, I imagine we'll also want to update:
`docs/platforms/unity/metrics/index.mdx`

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 1]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
     // Your code goes here
 }
 ```
+
+# OpenTelemetry Metrics
+
+The Sentry SDK for .NET also allows you to configure OpenTelemetry metrics that you would like to automatically capture and send to Sentry.
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:103]:
```suggestion
You can configure the .NET SDK to automatically capture and send OpenTelemetry metrics to Sentry.
```

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 2]
```diff
@@ -14,6 +12,8 @@ Metrics for .NET are supported with Sentry .NET SDK version `4.0.0` and above.
 
 Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
 
+# Custom metrics
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:15]:
```suggestion
# Custom Metrics

Sentry comes with built-in metrics, such as performance metrics and web vitals. You can also send custom metrics to Sentry to track the data points you care about most. This page describes how to set up and send custom metrics to use with our [Metrics feature](/product/metrics/). 
```

Wasn't sure if you meant to add more here as it's still a draft, but I think including an intro paragraph here to contextualize the rest of the info for this page would help.

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 3]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
     // Your code goes here
 }
 ```
+
+# OpenTelemetry Metrics
+
+The Sentry SDK for .NET also allows you to configure OpenTelemetry metrics that you would like to automatically capture and send to Sentry.
+
+For example, you could configure Sentry to capture all of the built-in metrics that it knows about:
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:105]:
```suggestion
For example, you can configure Sentry to capture all the built-in metrics that it knows about:
```

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 4]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
...
+
+```csharp
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:107]:
```suggestion
```csharp {13}
```
Not sure if we want to highlight line 13 here to call out how to capture all metrics

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 5]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
...
+    );
+
+builder.WebHost.UseSentry(options =>
+{
+    options.dsn = "___DSN___";
+    options.EnableMetrics(metrics => metrics.CaptureSystemDiagnosticsMeters = BuiltInSystemDiagnosticsMeters.All);
+});
+```
+
+Or to capture only metrics from specific Meters:
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:125]:
```suggestion
You can also capture only metrics from specific Meters:
```

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 6]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
...
+
+```csharp
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:127]:
```suggestion
```csharp {5-6}
```

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 7]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
...
+{
+    options.dsn = "___DSN___";
+    metrics.CaptureSystemDiagnosticsMeters = [
+        BuiltInSystemDiagnosticsMeters.MicrosoftAspNetCoreHosting,
+        BuiltInSystemDiagnosticsMeters.SystemNetHttp,
+    ];
+});
+```
+
+Or even to capture only specific Instruments:
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:138]:
```suggestion
Or, you can choose to capture only metrics from specific Instruments:
```

File: docs/platforms/dotnet/common/metrics/index.mdx [hunk 8]
```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
...
+
+```csharp
```
Comment (vivianyentran) [inline docs/platforms/dotnet/common/metrics/index.mdx:140]:
```suggestion
```csharp {11}
```

**Review by vivianyentran — APPROVED** [review]
Added some suggestions!

File: docs/platforms/dotnet/common/metrics/index.mdx (hunk 4 above)
Comment (jamescrosswell) [reply docs/platforms/dotnet/common/metrics/index.mdx:107]: Oh, that's awesome. I didn't know that was even possible 😁

**Comment by jamescrosswell** [comment]
Metrics are apparently going live soon

**Pull request closed without merging**